src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileStatusService.scala

```diff
@@ -55,10 +48,9 @@ object FileStatusService {
   val ChecksumMatch = "ChecksumMatch"
   val Antivirus = "Antivirus"
   val FFID = "FFID"
+  val Redaction = "Redaction"
```

Comment:
Could you update the unit tests to check this value, and update for the other values, as the unit tests on line 236, and 242 look out of date, as missing a few values.

Reply:
I've updated the unit tests in `FileStatusServiceSpec` to check all of the values.